public/app/plugins/panel/status-history/module.tsx

```diff
@@ -56,16 +56,6 @@ export const plugin = new PanelPlugin<StatusPanelOptions, StatusFieldConfig>(Sta
         },
         defaultValue: VisibilityMode.Auto,
       })
-      .addSliderInput({
```

Comment:
the property was not used in the panel. The slider existed in the options but did nothing. My guess is that it was a remnant of the `TimelineChart` interface. I think some props from one panel are remnants from the other. Both StateTimeline and StatusHistory had `rowHeight` and `alignValue` props which were also props in `TimelineChart`, but only StateTimeline actually had the options.

Reply:
For me it looks like it is working. I also think that align value would make sense here as well. @mdvictor 

https://user-images.githubusercontent.com/13729989/215051883-19a44df2-9f9b-4882-be2e-84300587ce8e.mov

Reply:
Oh my, that's correct. That was a blunder from my part. I opened this PR to readd the functionality here: https://github.com/grafana/grafana/pull/62293

As for the alignValue, that would mean a bit more implementation and modification of the `TimelineChart` component. May be better to open an enhancement ticket for that one.

Thank you for pointing this out @zoltanbedi ! It would've been bad to release the panel without that option :S